frontend/src/lib/user.ts

```diff
@@ -94,6 +94,7 @@ function jwtToUser(user: JwtTokenUser): LexAuthUser {
 
 export function logout(cookies?: Cookies): void {
   cookies && cookies.delete('.LexBoxAuth')
+  removeAllNotifications();
```

Comment:
I think this needs to be behind the `if (browser)` condition, like the line below.

Reply:
Actually, maybe I'm wrong. I'm not sure.

Reply:
As discussed, it's probably fine outside of the browser condition, so that's probably better and it's simpler.